src/xml.tsx

```diff
@@ -272,6 +272,13 @@ export function parse(source: string, middleware?: Middleware): JsxAST | null {
     );
   }
 
+  function decodeEntities(text: string): string {
+    return text
+      .replace(/&amp;/g, '&')
+      .replace(/&lt;/g, '<')
+      .replace(/&gt;/g, '>');
```

Comment:
Additionally, what about numeric character references? According to the spec, it's also a valid text content. Example:`&lt;` can be represented as `&#60;`, `&gt;` as `&#62;` and `&amp;` as `&#38;`